src/rgw/rgw_log.cc

```diff
@@ -128,14 +128,13 @@ class UsageLogger {
     round_timestamp = ts.round_to_hour();
   }
 
-  void insert_user(utime_t& timestamp, const rgw_user& user, rgw_usage_log_entry& entry) {
+  void insert_user(utime_t& timestamp, const string& user, rgw_usage_log_entry& entry) {
```

Comment:
`rgw_user` didn't contain the subuser info, the `insert_user` internal use the `rgw_user.to_str()` ultimately.So we need to  concatenate the `user_name:subuser_name`